public/static/js/menu.js

```diff
@@ -14,6 +14,10 @@ if (typeof cssmenu_no_js === 'undefined') {
         if (!isHamburger) {
             open_main_item = container.classList.contains('open') ? container: null;
         }
+
+		if (container.className.indexOf("expand-container")) {
+			return;
+		}
```

Comment:
this is required, s.t. the link clicks on "Downloads" and "Packages" (i.e. no sections) don't get silently swallowed.